src/Core/Components/Slider/FluentSlider.razor

```diff
@@ -0,0 +1,28 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@inherits FluentInputBase<TValue>
+@typeparam TValue
+
+<FluentField InputComponent="@this" ForId="@Id" Class="@ClassValue" Style="@StyleValue">
+	<fluent-slider slot="input"
+	               id="@Id"
+	               disabled="@Disabled"
+	               min="@FormatValueAsString(Min)"
```

Comment:
These attributes must return a `null` value if the parameter is null, using the new method `FormatValueAsStringOrNull`
```razor
min="@FormatValueAsStringOrNull(Min)"
max="@FormatValueAsStringOrNull(Max)"
value="@FormatValueAsString(Value)"
step="@FormatValueAsStringOrNull(Step)"
```